README.md

```diff
@@ -8,6 +8,11 @@ This component provides a tcp server which awaits a message on port 1234 with a
 
 In the configuration, the EEPROM Read feature can be enabled and disabled. If the feature is enabled, the tcp server sends the content of a external EEPROM to the connected client directly after the connection was accepted and before the client shall send the URL. The EEPROM is connected via I2C with the ESP32(S2). The I2C connection can be configured in the configuration menu.
 
+## Task Watchdog
+The ota update server is monitored by a task watchdog, which has to be triggered within a certain time. The allowed time window can be adjusted via the setting CONFIG_ESP_TASK_WDT_TIMEOUT_S in the configuration menu. If the ota update server is in standby and waits for a connecting client, it triggers the watchdog every 3 seconds.
+
+ATTENTION: During the update the watchdog is not triggered. How long this may take depends on the size of the image. This time must be considered when configuring the watchdog timeout.
```

Comment:
The maximum image size is known (from the partition table). The default timeout shall be set to allow flashing of one OTA partition.